content/en/continuous_integration/tests/setup/java.md

```diff
@@ -26,17 +26,23 @@ further_reading:
 ## Compatibility
 
 Supported test frameworks:
-* JUnit >= 4.10 and >= 5.3
-* TestNG >= 6.4
-* Spock >= 2.0
-* Cucumber >= 5.4.0 (earlier versions have limited support)
 
-Supported build systems:
-* Gradle >= 2.0
-* Maven >= 3.2.1
+| Test Framework | Version |
+|---|---|
+| JUnit | >= 4.10, 5.3 |
+| TestNG | >= 6.4 |
+| Spock | >= 2.0 |
+| Cucumber | >= 5.4.0 |
```

Comment:
Since this section is being changed, could you please add `Karate | >= 1.0.0` (support for this framework was added recently).

Reply:
```suggestion
| Cucumber | >= 5.4.0 |
| Karate | >= 1.0.0 |
```